### PR TITLE
Bxmsdoc 3423

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc [hunk 1]
```diff
@@ -5,7 +5,9 @@ You can view the assigned shifts for particular employees in an employee-centric
 
 .Procedure
 . To view a table of employees and shifts, click the *Availability Roster* tab.
-. In the top right part of the browser window, the *Week of* field lists the week for which the roster is displayed. To view other weeks, use the *<* and *>* buttons next to the *Week of* field. Alternatively, click on the date field and change the date to view the week that includes this date.
-. If the list is longer than 10 employees, the numbers of currently visible employees are displayed in the top right part of the browser window, for example, `1-15 of 34`. Use the *<* and *>* buttons next to the numbers to display other employees in the list. 
++
+In the top left part of the window, you can see the dates for which the roster is displayed. To view other weeks, you can use the *<* and *>* buttons next to the *Week of* field. Alternatively, you can click on the date field and change the date to view the week that includes this date.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc]: As B4

Here I would make To view other weeks, use.... it's on steps. The put a note for that step:
Alternatively, click the 
Click, never click on. Search for click on.

File: doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc [hunk 2]
```diff
@@ -5,7 +5,9 @@ You can view the assigned shifts for particular employees in an employee-centric
...
++
+You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-10 of 34`. You can use the *<* and *>* buttons next to the numbers to display other employees in the list.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc]: As b4 Look for this pattern throughout. Occurs several times below.

File: doc-content/enterprise-only/employee-rostering/er-tenants-con.adoc [hunk 3]
```diff
@@ -3,5 +3,5 @@
 
 The application supports multiple _tenants_. Each tenant is a completely independent set of data, including inputs and roster outputs. Changing data for one tenant does not affect other tenants. You can switch between tenants to use several independent data sets, for example, to prepare employee rosters for different locations.
 
-Several sample tenants are already present after installation, representing several typical enterprise types, such as a factory or hospital. You can select any of these tenants and modify them to suit your needs. You can also create a new tenant to start entering data from a blank slate.
+Several sample tenants are already present after installation, representing several typical enterprise types such as a factory or hospital. You can select any of these tenants and modify them to suit your needs. You can also create a new tenant to start entering data from a blank slate.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/er-tenants-con.adoc]: tenants are present (remove already)
...also create a new tenant to enter data from...

File: doc-content/enterprise-only/employee-rostering/optashift-ER-openshift-deploy-con.adoc [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
+You can deploy the {PLANNER} Employee Rostering starter application to {OPENSHIFT} using an OpenShift template or using the `provision.sh` shell script that is provided in the add-ons distribution. 
 
-The application source code is built and packaged locally and uploaded to the OpenShift environment for deployment.  Use this preferred method when Java developer tools (Java Development Kit and Maven) and a bash shell command line are available. 
+If you use an OpenShift template, {OPENSHIFT} loads an image with the application from the Red Hat repository and deploys it. This method of deployment is a technological preview.
 
-IMPORTANT: The data is held in memory and lost when the pod is restarted.
+If you use the script, it builds and packages the application source code locally and uploads it to the OpenShift environment for deployment.  You can use this method when Java developer tools (Java Development Kit and Maven) and a bash shell command line are available. 
+
+IMPORTANT: In the deployed application, the data is held in memory and lost when the pod is restarted.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-openshift-deploy-con.adoc]: Consider using the note form:
[IMPORTANT
====
BLah
====

File: doc-content/enterprise-only/openshift/registry-auth-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
+[id='registry-auth-proc']
+= Configuring OpenShift registry authentication
+
+You must configure OpenShift registry authentication to access Red Hat product images.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/registry-auth-proc.adoc]: Maybe add a little more here, like After you do BLAH, you must cofigure..

And I would put the next line in a note.

Any prereqs? Like You must have BLAH installed.. Whatever...

File: doc-content/enterprise-only/openshift/registry-auth-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,32 @@
+[id='registry-auth-proc']
+= Configuring OpenShift registry authentication
+
+You must configure OpenShift registry authentication to access Red Hat product images. 
+
+This procedure is not required if you are using a Red Hat OpenShift Online account.
+
+.Procedure
+
+. Complete the steps documented in https://access.redhat.com/RegistryAuthentication#registry-service-accounts-for-shared-environments-4[Registry Service Accounts for Shared Environments]. You must log on to Red Hat Customer Portal to access the document and to complete the steps. You must create a registry service account.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/registry-auth-proc.adoc]: See my comment in the other PR.

File: doc-content/enterprise-only/employee-rostering/er-employees-proc.adoc [hunk 7]
```diff
@@ -6,7 +6,9 @@ You must enter the list of all employees of the business and the skills they pos
 .Procedure
 
 . To enter or change the list of employees, click the *Spots* tab.
-. If the list of longer than 10 employees, the numbers of currently visible employees are displayed in the top right part of the browser window, for example, `1-15 of 34`. Use the *<* and *>* buttons to display other employees in the list. 
++
+You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other employees in the list.
```
[the remark this answers is not in the document]
Comment (mramendi) [reply doc-content/enterprise-only/employee-rostering/er-employees-proc.adoc]: Leaving this one post GA as I'm not at all sure about adding such a step, and I have already put this into the "needs guidance" document so it's traceable.

File: doc-content/enterprise-only/employee-rostering/er-employees-proc.adoc [hunk 8]
```diff
@@ -6,7 +6,9 @@ You must enter the list of all employees of the business and the skills they pos
 .Procedure
 
 . To enter or change the list of employees, click the *Spots* tab.
-. If the list of longer than 10 employees, the numbers of currently visible employees are displayed in the top right part of the browser window, for example, `1-15 of 34`. Use the *<* and *>* buttons to display other employees in the list. 
++
+You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other employees in the list. 
++
 . To add a new employee:
```
[the remark this answers is not in the document]
Comment (mramendi) [reply doc-content/enterprise-only/employee-rostering/er-employees-proc.adoc]: Added this to "guidance needed" but made the change for now.

File: doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc (hunk 1 above)
Comment (mramendi) [reply doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc]: Leaving steps for post GA as previous, but removing click on.

File: doc-content/enterprise-only/openshift/registry-auth-proc.adoc (hunk 5 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/registry-auth-proc.adoc]: It's a prereq itself. There is no "after you do BLAH", the only prereq for this is OpenShift itself, which is a prereq for the entire doc to start with.

**Pull request merged**